src/main/java/wolox/training/models/Book.java

```diff
@@ -34,11 +40,26 @@
     @Column(nullable = false)
     private String year;
 
-
     public Book() {
         // Added to use with JPA;
     }
 
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
```

Comment:
We need to protect the integrity of the id at all cost, **"setId"** puts the integrity of the id at risk, please delete it.